scripts/release/publish-docs-content.sh

```diff
@@ -52,17 +53,20 @@ do
   fi
 done
 
-# Move guide files over to $repoPath/guides
+# Copy guide files over to $repoPath/guides
 for filename in $overviewFiles*
 do
   if [ -f $filename ]; then
     cp -r $filename $repoPath/guides
   fi
 done
 
-# Move highlighted examples into $repoPath
+# Copy highlighted examples into $repoPath
 cp -r $examplesSource/* $repoPath/examples
 
+# Copy example plunker assets
+rsync -a $docsPath/plunker $repoPath/plunker
```

Comment:
I don't think we'll ever need this running on a Window machine since it's not part of the dev workflow.